### PR TITLE
Add wolfSSL_GENERAL_NAME_print

### PR DESCRIPTION
**Pull request by TakayukiMatsuo**
- GENERAL_NAME_print

### CODE REVIEW

File: src/ssl.c [hunk 1]
```diff
@@ -20217,6 +20217,124 @@ void wolfSSL_GENERAL_NAMES_free(WOLFSSL_GENERAL_NAMES *gens)
     wolfSSL_sk_free(gens);
 }
 
+#ifdef OPENSSL_ALL
+int wolfSSL_GENERAL_NAME_print(WOLFSSL_BIO* out, WOLFSSL_GENERAL_NAME* gen)
```
Comment (cconlon) [inline src/ssl.c:20221]: Please add a function comment describing behavior, arguments, and return values, thanks

File: src/ssl.c [hunk 2]
```diff
@@ -44954,7 +45072,7 @@ void wolfSSL_sk_X509_NAME_free(WOLF_STACK_OF(WOLFSSL_X509_NAME)* sk)
 
     XFREE(sk, sk->heap, DYNAMIC_TYPE_OPENSSL);
 }
-
+#if 1 /*------------------------test ------------------------------*/
```
Comment (cconlon) [inline src/ssl.c:45075]: Looks like a test/debug line got left in here?  Please remove.

File: src/ssl.c [hunk 3]
```diff
@@ -45110,7 +45228,7 @@ int wolfSSL_X509_NAME_print_ex_fp(XFILE file, WOLFSSL_X509_NAME* name,
     return ret;
 }
 #endif /* NO_FILESYSTEM */
-
+#endif /* ----------------------test ----------------------- */
```
Comment (cconlon) [inline src/ssl.c:45231]: Looks like a test/debug line got left in here?  Please remove.

File: src/ssl.c (hunk 1 above)
Comment (TakayukiMatsuo) [reply src/ssl.c:20221]: Added comments.

File: src/ssl.c (hunk 2 above)
Comment (TakayukiMatsuo) [reply src/ssl.c]: Removed the line.

File: src/ssl.c (hunk 3 above)
Comment (TakayukiMatsuo) [reply src/ssl.c]: Removed the line.

**Review by cconlon — APPROVED** [review]

**Pull request merged**